app/assets/sass/application-common.scss

```diff
@@ -74,8 +74,27 @@ li.task-item {
   padding-top: em(35);
   }
 
-  div.declaration-header {
+div.declaration-header {
   text-align: center;
   margin-top: 5px;
   margin-bottom: 5px;
+}
+
+#global-header .header-proposition #sign-out-header {
+  @include bold-16;
+  margin-top: 4px;
+  float: right;
+  color: $white;
+  text-decoration: none; }
+  @media (min-width: 641px) {
```

Comment:
GDS have a `tablet` variable that will handle the `min-width: 641px` so we'd be better to use that because if they change the width in future we'd get those changes automatically. For example:

`@media(tablet)`